### PR TITLE
test: make sure process killing logic works without `ps` on Linux

### PR DESCRIPTION
**Pull request by aslushnikov**
The bare-bones `debian` distribution we use for testing doesn't have
`ps`. This patch switches to reading `/proc` file system directly on
Linux instead of relying on `ps`.

Performance measurements for the 20000 active processes on Debian Docker
container, tested on my M1 Max Mac:
- the `ps -eo pid,ppid,pgid` call + parsing takes 293ms
- the manual synchronous `/proc` traversal + parsing takes 326ms

So this is ~10% perf penalty.

Drive-by: rename `pgid` into `pgrp` so that it stands out from `pid`
(process ID) and `ppid` (parent process ID).

### CODE REVIEW

File: tests/config/commonFixtures.ts [hunk 1]
```diff
@@ -32,28 +33,59 @@ import childProcess from 'child_process';
...
+      const match = statFile.match(/^(?<pid>\d+)\s+\((?<comm>.*)\)\s+(?<state>R|S|D|Z|T|t|W|X|x|K|W|P)\s+(?<ppid>\d+)\s+(?<pgrp>\d+)/);
+      if (match) {
+        result.push({
+          pid: +match.groups.pid,
+          ppid: +match.groups.ppid,
+          pgrp: +match.groups.pgrp,
+        });
+      }
+    } catch (e) {
+      // We don't have access to some /proc/<pid>/stat file.
```
Comment (yury-s) [inline tests/config/commonFixtures.ts:58]: should we log this for easier debugging?

**Review by yury-s — APPROVED** [review]

File: tests/config/commonFixtures.ts (hunk 1 above)
Comment (aslushnikov) [reply tests/config/commonFixtures.ts:58]: I don't think we should: all the descendants should be accessible, and all the other processes (e.g. root-spawned) might happen regularly. This is not an edge-case.
